detect_secrets/main.py

```diff
@@ -164,5 +171,30 @@ def _read_from_file(filename):  # pragma: no cover
         return json.loads(f.read())
 
 
+def _get_exclude_files(old_baseline):
+    """
+    Older versions of detect-secrets always had an `exclude_regex` key,
+    this was replaced by the `files` key under an `exclude` key in v0.12.0
+
+    :rtype: str|None
+    """
+    if old_baseline.get('exclude'):
+        return old_baseline['exclude']['files']
+    if old_baseline.get('exclude_regex'):
+        return old_baseline['exclude_regex']
+
+
+def _add_baseline_to_exclude_files(args):
+    """
+    Modifies args.exclude_files in-place.
+    """
+    baseline_name_re = r'^{}$'.format(args.import_filename[0])
```

Comment:
Please change naming convention from `[\w+]_re` to `[\w+]_regex`. It's easier to read, and unless there's a specific reason why the abbreviation is used, we can spare the extra three characters. :)

Reply:
I saw Anthony Sottile use `_re` on a live stream and have been doing it since lol, I'll be sure to change it not here but other places too.